### PR TITLE
V2 overview page

### LINKED ISSUE #940: watch only wallet menu hidden
If a watch only wallet is the only wallet in the list, the menu for the wallet doesn't show(hidden at the top right corner), and the scroll bar keeps showing, even if there's nothing to scroll to

Screenshot from 2022-05-22 21-11-24

### LINKED ISSUE #985: V2 overview page redesign
Moving to the single wallet design, the overview page is not the new overview page.

image

https://www.figma.com/file/e0cYaAzkXyXXPY00v23cQY/GoDCR-%26-DCRMobile?node-id=3559%3A131307

### LINKED ISSUE #1003: clicking the backdrop while trying to create a new wallet crashes the app
Screenshot from 2022-06-22 18-15-12

From the picture above, the modal presented is for creating a new wallet. Clicking the backdrop or clicking the cancel button causes the app to crash.

See log below:
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x11f6554]

goroutine 15 [running]:
github.com/planetdecred/godcr/ui/modal.(*CreatePasswordModal).Handle(0xc002984400)
        /home/kennedy/Projects/src/github.com/dreacot/godcr/ui/modal/create_password_modal.go:219 +0x814
github.com/planetdecred/godcr/ui.(*Window).displayWindow(0xc00015c000, {{0xc0a4f184141df208, 0x20f6f2f8f5, 0x2f31380}, {0x3f800000, 0x3f800000}, {0x320, 0x2bc}, {0x0, 0x0, ...}, ...})
        /home/kennedy/Projects/src/github.com/dreacot/godcr/ui/window.go:176 +0x10e
github.com/planetdecred/godcr/ui.(*Window).HandleEvents(0xc00015c000)
        /home/kennedy/Projects/src/github.com/dreacot/godcr/ui/window.go:148 +0x156
main.main.func2()
        /home/kennedy/Projects/src/github.com/dreacot/godcr/main.go:80 +0x26
created by main.main
        /home/kennedy/Projects/src/github.com/dreacot/godcr/main.go:79 +0x4d2
exit status 2
```

### PR DESCRIPTION
**Pull request by Sirmorrison**
Fix #985 
Fix #962 
Fix #940
Fix #1003

# first v2 design implementation
image

# Updated v2 design implementation
image

image

image

image

### CODE REVIEW

File: ui/decredmaterial/progressbar.go [hunk 1]
```diff
@@ -152,11 +176,8 @@ func (mp *MultiLayerProgressBar) progressBarLayout(gtx C) D {
 	}
 
 	calProgressWidth := func(progress int) int {
-		val := 1
-		if mp.total != 0 {
-			val = (progress / mp.total) * 100
-		}
-		return (mp.Width / 100) * val
+		val := (progress / mp.total) * 100
```
Comment (JustinBeBoy) [inline ui/decredmaterial/progressbar.go]: why you  don't check mp.total values, I have got case mp.total = 0

File: ui/page/info/info_page.go [hunk 2]
```diff
@@ -0,0 +1,317 @@
...
+	remainingSyncTime    string
+	syncStepLabel        string
+	headersToFetchOrScan int32
+	stepFetchProgress    int32
+	syncProgress         int
+	syncStep             int
+	isBackupModalOpened  bool
+}
+
+func NewInfoPage(l *load.Load) *WalletInfo {
```
Comment (JustinBeBoy) [inline ui/page/info/info_page.go:76]: why change the name of the page to info but keep the id as "Overview"?

**Review by JustinBeBoy — CHANGES_REQUESTED** [review]

**Comment by JustinBeBoy** [comment]
**- The progress didn't work**


**- Title tab not show**


**- Sync progress showing wrong**


**- Wallet info not show**
aaaaa

**Comment by Sirmorrison** [comment]
image

image

resolved progress bar bug..

**Review by JustinBeBoy — APPROVED** [review]

**Review by dreacot — COMMENTED** [review]
LGTM

**Pull request merged**